Makefile

```diff
@@ -12,12 +12,10 @@ clean:
 
 .PHONY: images
 images:
-	@docker build -f Dockerfiles/centos5 -t $(IMAGE)/centos5 .
 	@docker build -f Dockerfiles/centos6 -t $(IMAGE)/centos6 .
 	@docker build -f Dockerfiles/centos7 -t $(IMAGE)/centos7 .
 
 .PHONY: tests
 tests:
-	@docker run --rm -v $(shell pwd):/data:Z $(IMAGE)/centos5 ./run_unit_tests.sh
```

Comment:
These is already in PR #75.

Reply:
Yes, but that one is not merged yet. Without this change, the tests would be failing.

Reply:
It's not a problem, I'll resolve the conflict either here or in #75, based on whichever PR will be merged first.